packages/ui-components/src/Input/CheckboxInput.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import classNames from "classnames/bind";
+import { CommonInputProps, CommonInput } from "./CommonInput";
+
+interface CheckboxProps {
```

Comment:
Okay - updated. This also caused there to be a new input type, multiline, which differs from single line text inputs because multiline renders a <textarea />, and thus accepts textarea props, which differ from <input /> props single line text inputs render.

Reply:
CommonInputProps will likely not need to extend input props, because it accepts an input component (such as text, or password) as a prop. the props CommonInput uses are only for styling the help / error messages, which is an internal convention, not related to native inputs, I believe...

Text, Checkbox have been updated to extend the native <input /> props.